tests/trainer/test_trainer.py

```diff
@@ -3063,6 +3064,8 @@ def test_end_to_end_example(self):
                 "--predict_with_generate",
                 "--ddp_timeout",
                 "60",
+                "--report_to",
+                "none",
```

Comment:
Since `Codecarbon` is not supported on ROCm, the report callbacks are skipped from the trainer tests. Ref: https://github.com/huggingface/transformers/pull/30266